config.lua

```diff
@@ -45,6 +45,9 @@ Config.Locations = {
     ["shop"] = {
         coords = vector4(1786.19, 2557.77, 45.62, 0.5),
     },
+    ["reclaim"] = {
+        coords = vector4(1837.94, 2593.11, 45.89, 76.94),
```

Comment:
Why make this even a vector4? a vector3 would make your code above much cleaner.

Reply:
I'm sorry, it looks like it was a mistake

Reply:
Can you explain to me what the goal is when you press resovle conversation 
I did not understand the purpose of it